Resources/Prototypes/_Pirate/Entities/Structures/Doors/Airlocks/access.yml

```
@@ -0,0 +1,120 @@
+      board: [ DoorElectronicsIPOfficer ]
+      
+# External Syndicate Dune
+- type: entity
+  parent: AirlockShuttleSyndicate
+  id: AirlockExternalShuttleInterdyneLocked
+  suffix: External, Docking, Interdyne, Locked
+  components:
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronicsIP ]
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)